src/main/java/mate/controller/driver/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,31 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = {"/drivers/currentcars"})

```

Comment:
```suggestion
@WebServlet(urlPatterns = "/drivers/cars")

```

It seems it's more readable this way

Reply:
ok